FueledUtils/UIExtensions.swift

```diff
@@ -75,7 +93,17 @@ public extension UILabel {
 }
 
 public extension UIActivityIndicatorView {
+	@available(*, deprecated, renamed: "animating")
 	public var fueled_animating: Bool {
+		get {
+			return self.animating
+		}
+		set {
+			self.animating = newValue
+		}
+	}
+
+	@nonobjc public var animating: Bool {
```

Comment:
Maybe keep the same name here?

Reply:
should be isAnimating, right?

Reply:
It should, but `isAnimating` is already taken by `UIActivityIndicatorView` :| The only solution I see would be to create an Objective-C wrapper, but I'm not sure how desirable that is